web/webplatform/fetch.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+VERSION="$(cat VERSION)"
+FILENAME="autokitteh-web-v${VERSION}.zip"
+
+echo "want version ${VERSION}"
+
+if [[ -r ${FILENAME} ]]; then
+    echo "up to date"
+    exit 0
+fi
+
+echo "not downloaded yet."
+
+PREV="$(ls -1 autokitteh-web-v*.zip > /dev/null 2>&1 || true)"
+
+echo "fetching ${FILENAME}..."
+
+curl -sL "https://github.com/autokitteh/web-platform/releases/download/v${VERSION}/${FILENAME}" > "${FILENAME}"
```

Comment:
Can you also read and check the file's hash?

Reply:
doesn't look like we publish the hash on the release, we'll have to do that. @RonenMars ?

Reply:
It's possible, and I'll implement it on my side.

Do we want the hash to be part of the file name or a separate text file?

Reply:
Separate text file. see https://github.com/temporalio/temporal/releases/tag/v1.25.1 for example, file `checksums.txt` which is the standard i think